android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -68,6 +78,88 @@
   Map<String, Object> constants;
   AsyncTask<Void, Void, Map<String, Object>> futureConstants;
 
+  private boolean isDebug = true;
+
+  private boolean isTelephony = true;
+
+  private boolean isCheckPackage = true;
+
+  private List<String> mListPackageName = new ArrayList<>();
+
+  private static final String IP = "10.0.2.15";
```

Comment:
what is this 'IP' ?

Reply:
it is part of `checkIp()` method to detecting virtual machine